### PR TITLE
feature: download and properly stream zipped files to the log download. TM-1324

### PR DESCRIPTION
**Pull request by rtirserio**
Unzips a compressed log file and streams contents as the response

### CODE REVIEW

**Comment by mjoyce91** [comment]
This looks good, but we'll need to update the `GET /api/v1/logs/{string}/` route, as it ignores the extension, which is a problem if there's a `test.log` and `test.zip`.

**Comment by rtirserio** [comment]
@mjoyce91 updated to allow the url pattern to accept the extension

File: talentmap_api/log_viewer/services.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import requests
 import logging
 import os
+from zipfile import ZipFile, is_zipfile
```
Comment (mjoyce91) [inline talentmap_api/log_viewer/services.py]: One more question before we merge - is this library already included, or will we need to push this in the deployment?

Comment (rtirserio) [reply talentmap_api/log_viewer/services.py]: its included already

**Pull request merged**